install/kubernetes/cilium/values.yaml

```diff
@@ -444,6 +444,11 @@ encryption:
 endpointHealthChecking:
   enabled: true
 
+# -- Enable endpoint status
```

Comment:
nitpicking: maybe we could have the status documentation here as well (i.e. in addition to the cilium configmap) so that it can be seen in `README.md`?